### DISCUSSION TITLE
The history of the libvips name?

### DISCUSSION THREAD

**Discussion by boan-jfm** [libvips/libvips#3650] [General]
I am just curious, why is it called libvips, I guess it is library... something?

**Comment by jcupitt** [comment]
Hi, the wiki page has the history:

https://en.wikipedia.org/wiki/VIPS_(software)

tldr: lib (rary of the) vasari image processing system

**Reply by boan-jfm** [reply]
"Library of the Vasari Image Processing System"

>... It is John's ingenuity and programming skills which have made libVips so successful today.

Thank you 🙏

**Reply by jcupitt** [reply]
Ah embarrassing, I didn't write that heh. I'll edit it.
